hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-tests/src/test/java/org/apache/hadoop/yarn/server/MiniYARNCluster.java

```diff
@@ -815,7 +815,6 @@ protected synchronized void serviceInit(Configuration conf)
 
     @Override
     protected synchronized void serviceStart() throws Exception {
-
```

Comment:
Avoid? I would prefer to reduce churn for commits.

Reply:
Done!